airflow/www/templates/airflow/confirm.html

```diff
@@ -28,11 +28,17 @@ <h2>Wait a minute.</h2>
       <pre><code>{{ details }}</code></pre>
     {% endif %}
   </div>
-  <form method="POST">
+  {% if endpoint %}
+    <form method="POST" action="/{{ endpoint }}">
+  {% else %}
+    <form method="POST">
+  {% endif %}
     <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
     <input type="hidden" name="confirmed" value="true">
-    {% for name,val in request.form.items() if name != "csrf_token" %}
-      <input type="hidden" name="{{ name }}" value="{{ val }}">
+    {% for args_gen_dict in (request.args.items(), request.form.items()) %}
```

Comment:
@ashb `request.args != request.form`, so looping over both of them. Is there any other better way you can think of?

Reply:
https://flask.palletsprojects.com/en/2.0.x/api/?highlight=request#flask.Request.values is both of them combined in one :)